harvesters/src/main/java/org/fao/geonet/kernel/harvest/harvester/csw/Aligner.java

```diff
@@ -425,7 +424,7 @@ private void updateMetadata(RecordInfo ri, String id, Boolean force) throws Exce
         if (date == null && !force) {
             log.debug("  - Skipped metadata managed by another harvesting node. uuid:" + ri.uuid + ", name:" + params.getName());
         } else {
-            if (!force && !ri.isMoreRecentThan(date)) {
+            if (Boolean.TRUE.equals(!force) && !ri.isMoreRecentThan(date)) {
```

Comment:
Why not just use the primitive `boolean` over `Boolean` as a parameter here? AFAICT, force is never and should never be null, so this would simplify this

Reply:
@tobias-hotz I have no idea why was used `Boolean` instead of the primitive `boolean`. This change was a Sonarlint improvement, keeping the original code.

Reply:
Sure, it's fine then. I would suggest moving to `boolean` for parameters where it can never be null, so we can avoid the Boolean.TRUE.equals stuff for simplicity, but it's fine to keep it this way as well.